proposals/description/fip_82b.ts

```diff
@@ -0,0 +1,463 @@
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x4a4f013dcba6b46103e81e286782135c0dda175e82564e878ae500734753e55e', // FEI_MINT_ADMIN
+        '{feiDAOTimelock}'
+      ],
+      description: 'Grant feiDAOTimelock the FEI_MINT_ADMIN role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x6c9ecf07a5886fd74a8d32f4d3c317a7d5e5b5c7a073a3ab06c217e9ce5288e3', // TOKEMAK_DEPOSIT_ADMIN_ROLE
+        '{optimisticTimelock}'
+      ],
+      description: 'Grant optimisticTimelock the TOKEMAK_DEPOSIT_ADMIN_ROLE role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0xd387c7eec7161b3637e694ef5cf8f1a9e29bfd35135c86c9b540bebec4ef221a', // PCV_SAFE_MOVER_ROLE
+        '{tribalCouncilTimelock}'
+      ],
+      description: 'Grant tribalCouncilTimelock the PCV_SAFE_MOVER_ROLE role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0xdf6ce05acd28d71e96472375ef55c4a692f167af3ccda9500570f7373c1ba22c', // PCV_GUARDIAN_ADMIN_ROLE
+        '{tribalCouncilTimelock}'
+      ],
+      description: 'Grant tribalCouncilTimelock the PCV_GUARDIAN_ADMIN_ROLE role'
+    },
+    ////// Authorise new PCV Guardian and revoke old PCV Guardian ////////
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x0866eae1216ed05a11636a648003f3f62921eb97ccb05acc30636f62958a8bd6', // PCV_CONTROLLER
+        '{pcvGuardianNew}'
+      ],
+      description: 'Grant newPCVGuardian the PCV_CONTROLLER_ROLE'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x55435dd261a4b9b3364963f7738a7a662ad9c84396d64be3365284bb7f0a5041', // GUARDIAN
+        '{pcvGuardianNew}'
+      ],
+      description: 'Grant newPCVGuardian the GUARDIAN role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: [
+        '0x0866eae1216ed05a11636a648003f3f62921eb97ccb05acc30636f62958a8bd6', // PCV_CONTROLLER
+        '{pcvGuardian}'
+      ],
+      description: 'Revoke PCV_CONTROLLER from old pcvGuardian'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: [
+        '0x55435dd261a4b9b3364963f7738a7a662ad9c84396d64be3365284bb7f0a5041', // GUARDIAN
+        '{pcvGuardian}'
+      ],
+      description: 'Revoke GUARDIAN from old pcvGuardian'
+    }
+  ],
+  description: `
+    FIP_82b: Create, grant and make the TribalCouncil admin of various access control roles to allow
+    it to manage the protocol.
+
+    These roles allow the TribalCouncil to authorise pods with the necessary access control to operate 
+    distinct parts of the protocol.
```

Comment:
Please link the PR, snapshot and tribe forum post in the description

Reply:
Done, and fleshed out the description